cmd/oci-runtime-tool/generate.go

```diff
@@ -608,6 +613,22 @@ func parseRlimit(rlimit string) (string, uint64, uint64, error) {
 	return parts[0], uint64(hard), uint64(soft), nil
 }
 
+func parseNamespace(ns string) (string, string, error) {
+	parts := strings.SplitN(ns, ":", 2)
+	if len(parts) == 0 || parts[0] == "" {
+		return "", "", fmt.Errorf("invalid namespace value: %s", ns)
+	}
+
+	switch len(parts) {
+	case 1:
+		return parts[0], "", nil
+	case 2:
+		return parts[0], parts[1], nil
+	}
+
+	panic("should not be reached")
```

Comment:
So you want to return an error even though the actual line will never be hit? From my experience, people use `panic` in these situations because it's more clear that if you ever hit that line it's a programming error (not something that should happen normally).